src/groupByKey.ts

```diff
@@ -2,6 +2,15 @@ type GroupsMap<T> = {
   [key: string]: T[];
 };
 
-export function groupByKey(items, key) {
-  // write code here;
+export function groupByKey<T>(items: T[], key: string): GroupsMap<T> {
+  return items.reduce<GroupsMap<T>>((acc, item) => {
+    const keyValue = item[key] as unknown as string;
```

Comment:
```suggestion
    const keyValue = item[key].toString();
```

I think we can do like this